src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -838,3 +839,12 @@ def load_arguments(self, _):
         c.argument('analytics_query', options_list=['--analytics-query', '-q'], help="Query to execute over Log Analytics data.")
         c.argument('timespan', help="Timespan over which to query. Defaults to querying all available data.")
     # endregion
+
+    # region disk encryption set
+    with self.argument_context('disk-encryption-set') as c:
+        c.argument('disk_encryption_set_name', disk_encryption_set_name)
+        c.argument('key_url', help='URL pointing to a key or secret in KeyVault.')
+        c.argument('source_vault', help='Resource ID of the KeyVault containing the key or secret.')
```

Comment:
per cli command coding guideline,  does this source_vault support name of vault?

Reply:
based on code,  name is supported, pls update help message.

Reply:
Thanks for finding this issue. I'll fix it in help.

Reply:
seems not fixed yet

Reply:
Fixed